src/domain/participant/index.js

```diff
@@ -278,6 +278,8 @@ const addLimitAndInitialPosition = async (participantName, limitAndInitialPositi
     if (limitAndInitialPosition.initialPosition == null) {
       limitAndInitialPosition.initialPosition = Config.PARTICIPANT_INITIAL_POSTITION
     }
+    let payload = Object.assign({}, limitAndInitialPositionObj, {name: participantName})
+    await Utility.produceGeneralMessage(TransferEventType.NOTIFICATION, 'limit-adjustment', createLimitAdjustmentMessageProtocol(payload), Utility.ENUMS.STATE.SUCCESS)
```

Comment:
Please add enum and refer to it instead of using a string within the code here and also on the following lines: 460, 467. I would go with either limitEventAction.LIMIT_ADJUSTMENT or eventAction.LIMIT_ADJUSTMENT.

Reply:
done